### PR TITLE
fix: allow empty month in dateInput

### PR DESCRIPTION
**Pull request by andreapiras**
☝ This title will appear in the changelog - keep it meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint).

## 🖼 Context

User should not always be provided with a default choice for month in order not to pollute user data
## 🚀 Changes
This PR set month to none if no value is passed

 

## ✅ Checklist

- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

**Comment by gusso** [comment]
@andreapiras Considering the deployments are not working, could you post a few screenshots of what it looks like before you click it and while the select is open? Or a gif of it working?

**Comment by dozortsev** [comment]
@gusso it works, you just have to manually construct the link

💁‍♂️ https://transferwise.github.io/neptune-web/branch/DS-99-dateinput-firstmonth/components/inputs/DateInput

**Comment by gusso** [comment]
I think we should have a better default for the month placeholder, as it will most certainly almost never fit. What do you think of `Month`?

File: packages/docs/pages/_app.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ if (process.env.NODE_ENV !== 'production' && typeof window !== 'undefined') {
 class MyApp extends App {
   componentDidMount() {
     const { pathname } = Router;
-    if (pathname === '/') {
+    if (pathname === '/' || pathname === '/_error') {
```
Comment (helendurranttw) [inline packages/docs/pages/_app.js:27]: These changes should be in their own PR.

**Comment by helendurranttw** [comment]
When I load the page I see a flash where the month changes from 12 to December. Could you look into that @andreapiras ?
